.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,18 +7,28 @@ assignees: ''
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+# Description
 
-**Interested in Helping**
-Let us know if you are interested in helping, and in what capacity. [eg: I want to write code and create a pull request, I can provide test cases, I have a suggestion on how it can be done, No thanks]
+A clear and concise description of what the desired feature is. 
 
+Is your feature request related to an exiting  problem? Please describe it and/or provide an issue number.
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+# Proposed solutions
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+Do you have a solution, what would you recommend to fix it?
+If you don't have a solution what would you like the end result to be?
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+# Can you help?
+
+Let us know if you are interested in helping, and in what capacity. eg: I want to write code and create a pull request, I can provide test cases, I have a suggestion on how it can be done, etc
```

Comment:
```suggestion
Let us know if you are interested in helping, and in what capacity. e.g.: I want to write code and create a pull request, I can provide test cases, I have a suggestion on how it can be done, etc.
```